### PR TITLE
Add to cart

### PR DESCRIPTION
**Pull request by mvn-vyle-dn**
## Description
- Add product to cart
- Modify cart: quantity, delete a product out of cart
## Reference Requirements
- Task ID: TI-1
## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-hienpham** [comment]
LGTM

File: public/js/public/cart.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        $('.box-cart').addClass("active");
+        cart = [];
+    }
+    var img = $('#item-product-' + idProduct +' .item .item-img a img').attr('src');
+    var name = $('#item-product-' + idProduct +' .item .item-name a h2').text();
+    var price = $('#item-product-' + idProduct +' .item .item-name .price span').text();
+    var cartItem;
+    if(cart.length > 0){
+        var resultObject = searchById(idProduct, cart);
+        var cartItem;
```
Comment (at-quangtran) [inline public/js/public/cart.js]: khai báo trùng

File: public/js/public/cart.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        }
+    }else {
+    cartItem  = {id:idProduct, name:name, img:img, price:price, quanlity: 1};
+    cart.push(cartItem);
+    }
+    sessionStorage.setItem("cart", JSON.stringify(cart));
+    showCart(cart);
+    $('.box-cart').addClass("active");
+}
+function searchById(nameKey, myArray){
```
Comment (at-quangtran) [inline public/js/public/cart.js]: searchById(value, data)

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**